13/umbraco-cms/SUMMARY.md

```diff
@@ -20,6 +20,14 @@
   * [External login providers](reference/security/external-login-providers.md)
   * [Two-factor Authentication](reference/security/two-factor-authentication.md)
 
+* [Content Delivery API](reference/content-delivery-api/README.md)
+    * [Custom property editors support](reference/content-delivery-api/custom-property-editors-support.md)
+    * [Extension API for querying](reference/content-delivery-api/extension-api-for-querying.md)
+    * [Media Delivery API](reference/content-delivery-api/media-delivery-api.md)
+    * [Protected content in the Delivery API](reference/content-delivery-api/protected-content-in-the-delivery-api.md)
+    * [Additional preview environments support](reference/content-delivery-api/additional-preview-environments-support.md)
```

Comment:
```suggestion

```

Reply:
@kjac This looks like the only article in this section with no changes, so I have suggested to remove it from the summary 😄

Reply:
@jonat123 are you sure about this? The article is in v13:

![image](https://github.com/umbraco/UmbracoDocs/assets/7405322/3d37b4cc-1dd7-48ba-9979-719508d1da46)

Reply:
Ah so it has to do with which ones that have changed? Thanks 😛 I have no idea how these summaries work across versions.